openedx/core/djangoapps/content/course_overviews/management/commands/generate_course_overview.py

```diff
@@ -33,12 +33,10 @@ class Command(BaseCommand):
 
     def handle(self, *args, **options):
 
+        course_keys = []
         if options['all']:
-            # Have CourseOverview generate course overviews for all
-            # the courses in the system.
-            CourseOverview.get_all_courses(force_reseeding=True)
+            course_keys = [course.id for course in modulestore().get_courses()]
```

Comment:
@nasthagiri I missed that this was merged.  Let's discuss offline to see if we want any follow-up.  My comment is about adding clarity, but isn't about correctness, so it shouldn't block anything.